README.md

```diff
@@ -11,7 +11,7 @@ The  README.md of the old repository is attached at the end of this file for you
 ## Contact
 As there are no specific support and contact channels for this project yet, please use the issue tracker of this repository to get in contact with the development community for the time being.
```

Comment:
"no specific support and contact channels" - should we take a better wording now?
Maybe also refer to FB or so?

Reply:
This readme only targets developers not primarily the community.

Within the app I referenced to FB already.